README.md

```diff
@@ -326,23 +308,81 @@ Open definition aside (default binding):
 nnoremap <C-w>gd <Cmd>call VSCodeNotify('editor.action.revealDefinitionAside')<CR>
 ```
 
-Run Find in files for word under cursor in vscode:
+Find in files for word under cursor:
 
 ```vim
 nnoremap ? <Cmd>call VSCodeNotify('workbench.action.findInFiles', { 'query': expand('<cword>')})<CR>
 ```
 
+More advanced examples can be found [here](https://github.com/vscode-neovim/vscode-neovim/tree/master/vim).
+
 ## ⌨️ Bindings
 
-**Custom keymaps for scrolling/window/tab/etc management**
+These are the default commands and bindings available for file/scroll/window/tab management.
 
--   See [vscode-scrolling.vim](/vim/vscode-scrolling.vim) for scrolling commands reference
--   See [vscode-file-commands.vim](/vim/vscode-file-commands.vim) for file commands reference
--   See [vscode-tab-commands.vim](/vim/vscode-tab-commands.vim) for tab commands reference
--   See [vscode-window-commands.vim](/vim/vscode-window-commands.vim) for window commands reference
+-   See [vscode-scrolling.vim](https://github.com/vscode-neovim/vscode-neovim/blob/master/vim/vscode-scrolling.vim) for
+    scrolling commands reference
+-   See
+    [vscode-file-commands.vim](https://github.com/vscode-neovim/vscode-neovim/blob/master/vim/vscode-file-commands.vim)
```

Comment:
is there some autoformatter doing this?

Reply:
Yeah, I had some issues with the autoformatter messing up with things, thanks for catching